### ISSUE TITLE
malloc error instantiating VisionReplayManager

### ISSUE THREAD

**Issue by avi-c** [mapbox/mapbox-vision-ios#308]
## Description
malloc error and crash trying to instantiate Vision replay with a particular recorded session. Session worked fine in vision 0.11.x and below.

Session can be found here: https://drive.google.com/open?id=1oeOTbyFBjDZ3xCS2caIOSVCvhbeHKssh

## Expected behavior
Replay begins as normal

## Actual behavior
malloc allocation error in ManagerDependencies.swift:47, in the call VisionReplayManagerNative.create. Error causes a crash.

## Configuration
 - Device: iPad 10.2
 - OS: iOS 13.3.1
 - Frameworks versions (leave the ones you're using):
   - MapboxVisionNative 0.12.0
   - MapboxVisionARNative 0.12.0
   - MapboxVision 0.12.0
   - MapboxVisionAR 0.12.0

**Comment by cisnayan** [comment]
Unexpectedly found nil while initialising the VisionReplayManager

let documentsPath =
            NSSearchPathForDirectoriesInDomains(.documentDirectory,
                                                .userDomainMask,
                                                true).first!
        let path = documentsPath.appending("/safety-alerts-drawing")

        // create VisionReplayManager with a path to recorded session
        visionManager = try? VisionReplayManager.create(recordPath: path)

**Comment by chezzdev** [comment]
Hey @cisnayan. Were you able to download the session and put it in the right place as described in the tutorial?

**Comment by cisnayan** [comment]
> Hey @cisnayan. Were you able to download the session and put it in the right place as described in the tutorial?

Yes @chezzdev I just follow the same code and try to run the same but when i debug i got to know that the visionManager is nil and crashed when we try to assign the delegate of it.
As the VisionReplayManager returning the nil object.

**Comment by chezzdev** [comment]
You need to download the recorded session and put it on the device. Please follow the instructions at https://docs.mapbox.com/help/tutorials/ios-vision-safety-alerts/#getting-started.
